ocpp-v1_6/src/main/java/eu/chargetime/ocpp/model/core/StartTransactionRequest.java

```diff
@@ -81,19 +82,19 @@ public void setConnectorId(Integer connectorId) throws PropertyConstraintExcepti
     /**
      * This contains the identifier for which a transaction has to be started.
      *
-     * @return the {@link IdToken}.
+     * @return the IdToken.
      */
-    public IdToken getIdTag() {
+    public String getIdTag() {
         return idTag;
     }
 
     /**
      * Required. This contains the identifier for which a transaction has to be started.
      *
-     * @param idTag the {@link IdToken}.
+     * @param idTag the IdToken.
      */
     @XmlElement
-    public void setIdTag(IdToken idTag) {
+    public void setIdTag(String idTag) {
```

Comment:
This method should validate the input and throw an exception if it's invalid. Fail fast. - This should be verified by test.